webapp/app/forms/steps/lotse/steuerminderungen.py

```diff
@@ -59,9 +59,10 @@ def render(self, **kwargs):
             form={
                 'action': self.render_info.submit_url,
                 'csrf_token': generate_csrf(),
-                'show_overview_button': bool(self.render_info.overview_url),
+                'show_overview_button': bool(self.render_info.overview_url)
```

Comment:
Out of curiosity: why did you delete the ','?

Reply:
It just annoyed me because it is the last element... :sweat_smile: